### ISSUE TITLE
prepit Errors

### ISSUE THREAD

**Issue by GoogleCodeExporter** [positioning/kalmanfilter#3]
```
What steps will reproduce the problem?
1.data(drifter)
2.prep.track<-prepit(drifter, fix.first=c(360-161.45,22.85,2002,9,10,0,0,0), 
fix.last=c(360-159.87,21.95,2003,5,21,0,0,0), scan=FALSE)  
3.

What is the expected output? What do you see instead?
Writing to temp files ... Done.
'prepro.exe' is not recognized as an internal or external command,
operable program or batch file.
Error in file(con, "r") : cannot open the connection
In addition: Warning messages:
1: '.path.package' is deprecated.
Use 'path.package' instead.
See help("Deprecated") 
2: '.path.package' is deprecated.
Use 'path.package' instead.
See help("Deprecated") 
3: '.path.package' is deprecated.
Use 'path.package' instead.
See help("Deprecated") 
4: running command 'C:\windows\system32\cmd.exe /c prepro.exe' had status 1 
5: In shell(cmd, invisible = TRUE) :
  'prepro.exe' execution failed with error code 1
6: In file(con, "r") :
  cannot open file 'ukf.dat': No such file or directory

What version of the product are you using? On what operating system?
trackit, Windows 7 64bit, R 3.0.3

Please provide any additional information below.
When using the prepit function (even with sample data/code), the outcome are 
these errors. Is there a fix?

```

Original issue reported on code.google.com by `shpate...@gmail.com` on 12 Mar 2014 at 5:36

**Comment by AndyAIG** [comment]
I'm still experiencing problems with the trackit function using the example 'drifter' data:
'ukf.exe' is not recognized as an internal or external command,
operable program or batch file.
Error in file(file, "rt") : cannot open the connection

Was this issue ever resolved?

Thanks, Andy

**Comment by tagtuna** [comment]
Hey Andy - thanks for bringing this issue up. It's strange, I suspect something has changed at the NOAA site. I will look into it, and see what I can do. If you need anything urgent, email me directly at tim.lam@umb.edu

**Comment by tagtuna** [comment]
Hi Andy, I just tracked down what's happening. I believe the NOAA ESRL subset API is being upgraded/ temporarily unavailable. The script we have in R basically recreates the web form processes, as if you were to request data manually. I tried in person to do it through the web form, but was experienced same issues. Feel free to file an issue with ESRL, as I am going to do so. My recommendation is to stay put for a few days (usually the problem goes away - been there before). Sorry about an external problem, beyond my control.
